test/test_log_to_cerr/test_log_to_cerr.cpp

```diff
@@ -27,7 +27,7 @@ int main(void)
     log.warning("Testing warning", true);
     log.error("Testing error", true);
     log.fault("Testing fault", true);
-    log.failur("Testing failur", true);
+    log.failure("Testing failur", true);
```

Comment:
`log.failure("Testing failure");`
a small correction of the spelling mistake in the string.